addons/event/views/event_event_views.xml

```diff
@@ -302,4 +318,22 @@
         <field name="action" ref="event.action_event_view"/>
     </record>
 
+    <record model="ir.ui.view" id="event_registration_view_form_inherit_barcode">
+        <field name="name">event.registration.view.form.inherit.barcode</field>
+        <field name="inherit_id" ref="event.view_event_registration_form"/>
+        <field name="model">event.registration</field>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='event_id']" position="after">
+                <field name="barcode" groups="base.group_no_one" />
+            </xpath>
+        </field>
+    </record>
+
+    <!-- EVENT.EVENT HEADER: REGISTRATION DESK MENU -->
+    <menuitem name="Registration Desk"
+        id="menu_event_registration_desk"
+        sequence="30"
+        action="event.event_barcode_action_main_view"
+        parent="event.event_main_menu"
+        groups="event.group_event_registration_desk"/>
```

Comment:
@thiryr i can't do this because things that it is reffering to are loaded second (if i move) therefore it raises an error because `event.event_barcode_action_main_view` isn't loaded yet

Reply:
If there's no dependency the other way around, you can change the order of the files in the manifest and it will fix your ordering issue. Otherwise fine I guess :shrug:

Reply:
@thiryr i re-ordered and it seems that there is another issue, so i don't think it's a big deal to leave it there